packages/patternfly-4/react-core/src/components/Pagination/Pagination.tsx

```diff
@@ -52,6 +52,22 @@ export interface PaginationTitles {
   paginationTitle?: string;
 }
 
+export type OnSetPage = (
+  _evt: React.MouseEvent | React.KeyboardEvent | MouseEvent,
+  newPage: number,
+  perPage?: number,
+  startIdx?: number,
+  endIdx?: number
+) => void;
+
+export type OnPerPageSelect = (
+  _evt: React.MouseEvent | React.KeyboardEvent | MouseEvent,
+  newPerPage: number,
+  newPage?: number,
```

Comment:
I was wondering why `newPage` needs to be optional? It's required in `OnSetPage` - is it possible to also make it required here?

Reply:
The original thought was that changing the **perPage** option would set a new **newPerPage** value but may not change the page - that said, the **newPage** argument is always provided by this function, updating it to required per your suggestion.